src/content/docs/guides/manual-installation.mdx

```diff
@@ -32,6 +32,33 @@ Biome is available as a [Homebrew formula](https://formulae.brew.sh/formula/biom
 brew install biome
 ```
 
+## Docker
+
+Biome publishes [official Docker images](https://github.com/biomejs/docker) that support 
+the **amd64** and **arm64** architectures for all Biome versions starting from `v1.7.0`.
```

Comment:
Perhaps this should link to https://github.com/biomejs/docker/pkgs/container/biome instead?

Reply:
Makes sense